doc/backend_vars.asciidoc

```diff
@@ -143,7 +143,7 @@ QEMUCPU;see qemu -cpu ?;undef;CPU to emulate
 QEMUCPUS;integer;1;Number of CPUs to assign to VM
 QEMUDIES;integer;undef;Number of CPU dies used by VM
 QEMUMACHINE;see qemu -machine ?;undef;Machine and chipset to emulate
-QEMUPORT;integer;20002 + worker instance * 10;Port on which QEMU monitor should listen
+QEMUPORT;integer;20003 + worker instance * 10;Port on which QEMU monitor should listen
```

Comment:
I think 20002 is actually correct, e.g. check the assignment in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Settings.pm. Note that the command server port is "+ 1" (see https://github.com/os-autoinst/os-autoinst/blob/master/OpenQA/Isotovideo/Runner.pm) but this is about `QEMUPORT` itself.